### PR TITLE
Rebuild Docker container for PR #76

### PR DESCRIPTION
**Pull request by drpatelh**
Initial PR to rebuild `Docker` container for https://github.com/nf-core/chipseq/pull/76

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,10 @@ FROM nfcore/base
 LABEL authors="phil.ewels@scilifelab.se" \
       description="Docker image containing all requirements for the nfcore/chipseq pipeline"
 
+## GAWK has the 'and' function, needed for run_spp.R script from phantompeakqualtools package
+RUN apt-get update && apt-get install -y gawk && apt-get clean -y
+RUN echo 'alias awk=gawk' >> ~/.bashrc
```
Comment (apeltzer) [inline Dockerfile]: Couldn't we use the recipe from here instead? 

https://anaconda.org/conda-forge/gawk

Comment (drpatelh) [reply Dockerfile]: Yep. Just spotted that :+1: :smile:

Comment (drpatelh) [reply Dockerfile]: Fixed.

**Comment by drpatelh** [comment]
Thanks @apeltzer . Ill merge when the tests pass.

**Pull request merged**